.jshintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "bitwise": false,
+  "freeze": true,
+  "globals": false,
```

Comment:
Causes "modules/servo-scanner.js: line 48, col 1, 'exports' is not defined. (W117)"